package.json

```diff
@@ -44,7 +44,7 @@
         "npm-package": "node ./tools/release-tools/npm-package.js"
     },
     "devDependencies": {
-        "@microsoft/rush": "^5.97.0",
+        "@microsoft/rush": "5.148.0",
```

Comment:
We need to update to at least version 5.133.1 or later for the internal builds to work.